typescript/web/src/components/members/new-member-modal.tsx

```diff
@@ -59,7 +59,10 @@ export const NewMemberModal = ({
     const inputValue = e.target.value;
     setValue(inputValue);
   };
-  const emails = value.split("\n").filter((email) => email !== "");
+  const emails = value
+    .split("\n")
```

Comment:
By using `.split(/[\s,;]+/)` (regex to be verified) you could remove the need to `.map` and also allow commas and semicolons.

Reply:
Done too. I checked the regex.